templates/index

```diff
@@ -146,7 +146,7 @@ cisco_asa_show_version.textfsm, .*, cisco_asa, sh[[ow]] ver[[sion]]
 cisco_asa_show_route.textfsm, .*, cisco_asa, sh[[ow]] ro[[ute]]
 cisco_asa_show_xlate.textfsm, .*, cisco_asa, sh[[ow]] x[[late]]
 cisco_asa_show_name.textfsm, .*, cisco_asa, sh[[ow]] nam[[e]]
-cisco_asa_show_arp.textfsm, .*, cisco_(asa|ftd), sh[[ow]] arp
```

Comment:
I just ran tox on master and it passed the tests, as well as the last build on here shows as passing. Removing this will cause it to not work for FTD devices. The test case is currently covered in ASA, as we do not require testing both platforms.

Reply:
No worries at all...will restore the index to its original state

FYI, here is the test failure for FTD (using `py.test -vv`)

```
tests/test_testcases_exists.py::test_verify_parsed_and_reference_data_exists[tests/vyatta_vyos/show_interfaces] PASSED [ 99%]
tests/test_testcases_exists.py::test_verify_parsed_and_reference_data_exists[tests/watchguard_firebox/show_arp] PASSED [100%]

============================================ FAILURES =============================================
_____________ test_verify_parsed_and_reference_data_exists[tests/cisco_ftd/show_arp] ______________

mock_directory = 'tests/cisco_ftd/show_arp'

    @pytest.mark.parametrize("mock_directory", extract_index_data())
    def test_verify_parsed_and_reference_data_exists(mock_directory):
        """Verify that at least one test exists for all entries in the index file."""
        cases = f"{mock_directory}/*.raw"
        test_list = glob.glob(cases)
>       assert len(test_list) != 0, f"Could not find tests for {mock_directory}.textfsm"
E       AssertionError: Could not find tests for tests/cisco_ftd/show_arp.textfsm
E       assert 0 != 0
E        +  where 0 = len([])

tests/test_testcases_exists.py:41: AssertionError
================================= 1 failed, 972 passed in 23.26s ==================================
```